site/templates/message.html

```diff
@@ -1,6 +1,10 @@
-{% extends "base.html" %}
-{% block title %}Send Brian a Message{% endblock title %}
-{% block body_content %}
+{% extends "base.html" %} {% block title %}Send Brian a Message{% endblock title %} {% block body_content %}
+
+<style>
+  .btn:not([disabled]) .spinner-grow {
+    display: none;
+  }
+</style>
```

Comment:
For showing and hiding stuff, instead of modifying the `style` attribute directly, I suggest using CSS to handle that.

With your case, you could either use classes and [classList.toggle](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList), or since you're already toggling `disabled` attribute, I created a CSS rule where the spinner is hidden if the button is not disabled.